components/brave_wallet_ui/utils/pricing-utils.ts

```diff
@@ -127,3 +128,21 @@ export const getPriceIdForToken = (
 
   return token.symbol.toLowerCase()
 }
+
+export const getPriceRequestForToken = (
+  token: Pick<
+    BraveWallet.BlockchainToken,
+    'coin' | 'chainId' | 'contractAddress'
+  >,
+) => {
+  // Skip price of testnet tokens
+  if (SupportedTestNetworks.includes(token.chainId)) {
+    return
+  }
```

Comment:
I'd rather move that to brave-core

Reply:
Now that we're passing chain id to gate3, this block no longer required (skipped within gate3).

07a8c23f77c3391b037cc4a270bb5c7e592f94ec